source/garbageHandler.py

```diff
@@ -20,7 +20,9 @@ class TrackedObject:
 	"""
 
 	def __del__(self):
-		notifyObjectDeletion(self)
+		# notifyObjectDeletion could be None if Python is shutting down.
+		if notifyObjectDeletion:
+			notifyObjectDeletion(self)

```

Comment:
Could you elaborate on this some more? Has this behavior changed with Python 3.8?

Reply:
this has not changed with Python 3.8. In fact you reported this bug to me many months ago. I only really noticed it properly as I was testing the rununittests.bat file. I can remove this commit and address it in a future pr if you like? Though now making unittests much easier and faster to run with the new command, that traceback is going to become more noticeable.